README.rst

```diff
@@ -118,8 +118,11 @@ dependencies on any operating system thanks to the
 `PyVISA-py <https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/>`__
 VISA backend. However, in order to use any VISA functionality that
 PyVISA-py does not implement, a third-party VISA backend such as
+`TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__
+or
 `NI-VISA <https://www.ni.com/en-us/support/downloads/drivers/download.ni-visa.html>`__
-can be installed. ``tm_devices`` will use an available VISA backend if
+can be installed. v4.2.0 is the TekVISA version supported by ``tm_devices``.
```

Comment:
The minimum version for TekVISA needs to moved above to be next to the TekVISA link

Reply:
```suggestion
can be installed. ``tm_devices`` will use an available VISA backend if
```

Reply:
Done.